src/components/Navbar/styles.scss

```diff
@@ -1,4 +1,4 @@
-.navbar {
+.dropdown--navbar {
```

Comment:
Why are you using double dashes in class names? Is there any special reason?

Reply:
There is no particular reason. I think it looks more clear but I will change it on later commits.